README.md

```diff
@@ -1,20 +1,41 @@
 # pipelines-api-examples
 
-This repository contains examples for the Pipelines API.
+This repository contains examples for the [Google Genomics Pipelines API]
+(https://cloud.google.com/genomics/reference/rest/v1alpha2/pipelines).
 
-The pipelines API is an easy way to create, run, and monitor command-line tools on Google Compute Engine in a Docker container. You can use it like you would a job scheduler. Input files are copied from a Cloud Storage bucket to a local disk, and output files are copied back to Cloud Storage. You can submit batch operations from your laptop, and have them run in the cloud, using any tools that you can package up with Docker, or that someone has already packaged up and shared.
+The API provides an easy way to create, run, and monitor command-line tools
+on Google Compute Engine running in a Docker container. You can use it like
+you would a job scheduler.
+
+The most common use case is to run an off-the-shelf tool or custom script
+that reads and writes files. You may want to run such a tool over files in
+Google Cloud Storage. You may want to run this independently over hundreds
+or thousands of files.
+
+The typical flow for a pipeline is:
+
+  1. Create a Compute Engine virtual machine
+  1. Copy one or more files from Cloud Storage to a disk
+  1. Run the tool on the file(s)
+  1. Copy the output to Cloud Storage
+  1. Destroy the Compute Engine virtual machine
```

Comment:
Hi Paul. Thanks for taking a look.

The markdown will be rendered as 1,2,3,4. I admit I go back and forth as to whether to use explicit numbering or implicit.

I'll typically only use explicit numbering if somewhere I need to say something like "in step 3 below", or if the actual text is complex enough that the numbering is likely to get broken.

I'm going to leave these as-is.